### DISCUSSION TITLE
长时间编辑文章，登录状态失效，无法保存文章

### DISCUSSION THREAD

**Discussion by AuntYang** [Qexo/Qexo#334] [General]
这几天完全把文章编辑工作迁移到Qexo上，但有一点比较困扰，Qexo自带图床的在线编辑挺好用，但长时间编辑，大约半小时？登录状态失效后，保存文章提示网络错误，这种情况只能是，在编辑时间过长的情况下，在编辑过程中不时地点击保存草稿，但写文章总是会忘记....有什么更好的办法吗？如果在本地编辑好，复制粘贴到Qexo上去，我感觉Qexo就失去了在线编辑的便利，而markdown在线编辑器也失去了意义。

**Comment by Fgaoxing** [comment]
我后面，打算搞一个随时存储，用cache实时存储，随时还原



在 2023-03-29 20:47:15，"AuntYang" ***@***.***> 写道：

这几天完全把文章编辑工作迁移到Qexo上，但有一点比较困扰，Qexo自带图床的在线编辑挺好用，但长时间编辑，大约半小时？登录状态失效后，保存文章提示网络错误，这种情况只能是，在编辑时间过长的情况下，在编辑过程中不时地点击保存草稿，但写文章总是会忘记....有什么更好的办法吗？如果在本地编辑好，复制粘贴到Qexo上去，我感觉Qexo就失去了在线编辑的便利，而markdown在线编辑器也失去了意义。

**Comment by am-abudu** [comment]
我设置的会话(Session)过期时间为24h，请检查你本地的环境是否有问题？比如某个浏览器设置或扩展清除了缓存？

**Reply by AuntYang** [reply]
![image](https://user-images.githubusercontent.com/49216233/228602951-583b530d-ad40-4747-bf94-ad6b3d8ee99c.png)
我明天用其他浏览器再试一下
